### PR TITLE
🎈 4.7.0

### PR DESCRIPTION
**Pull request by mjcarroll**
# 🎈 Release

Preparation for 4.7.0 release.

Comparison to 4.6.2: https://github.com/gazebosim/gz-common/compare/ignition-common4_4.6.2...ign-common4

## Checklist
- [ ] Asked team if this is a good time for a release
- [ ] There are no changes to be ported from the previous major version
- [ ] No PRs targeted at this major version are close to getting in
- [ ] Bumped minor for new features, patch for bug fixes
- [ ] Updated changelog
- [ ] Updated migration guide (as needed)
- [ ] Link to PR updating dependency versions in appropriate repository in [gazebo-release](https://github.com/gazebo-release) (as needed): <LINK>

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,5 +1,49 @@
...
+1. Port: 3 -> 4 (#506)
+    * [Pull request #506](https://github.com/gazebosim/gz-common/pull/506)
+
+1. Fix for ffmpeg v6
+    * [Pull request #497](https://github.com/gazebosim/gz-common/pull/497)
+
+1. Include cstring for memcpy
+    * [Pull request #501](https://github.com/gazebosim/gz-common/pull/501)
+
+1. Port: 3 -> 4 (#500)
```
Comment (azeey) [inline Changelog.md:14]: There are a number of 3->4 port PRs here. Can we consolidate them?

File: Changelog.md [hunk 2]
```diff
@@ -1,5 +1,49 @@
...
+1. Add marcoag as codeowner
+    * [Pull request #493](https://github.com/gazebosim/gz-common/pull/493)
+
+1. port: 3->4
+    * [Pull request #491](https://github.com/gazebosim/gz-common/pull/491)
+
+1. CI workflow: use checkout v3
+    * [Pull request #490](https://github.com/gazebosim/gz-common/pull/490)
+
+1. Update version in Changelog
```
Comment (azeey) [inline Changelog.md:35]: nit: We can probably skip this.

**Review by azeey — APPROVED** [review]

File: Changelog.md (hunk 1 above)
Comment (mjcarroll) [reply Changelog.md:14]: Whenever I start removing things from the changelog, @scpeters gets on to me ;)

Comment (azeey) [reply Changelog.md:14]: Oh, I didn't mean remove them. They could just be listed in a single heading.

**Review by ahcorde — APPROVED** [review]

File: Changelog.md (hunk 1 above)
Comment (mjcarroll) [reply Changelog.md]: Updated

File: Changelog.md (hunk 2 above)
Comment (mjcarroll) [reply Changelog.md]: removed

**Pull request merged**